arch/arm/boot/dts/broadcom/bcm2711-rpi-cm4.dts

```diff
@@ -498,13 +498,6 @@ i2c_csi_dsi0: &i2c0 {
 			<&ant2>, "output-high?=off",
 			<&ant2>, "output-low?=on";
 
-		cam0_reg = <&cam0_reg>,"status";
-		cam0_reg_gpio = <&cam0_reg>,"gpio:4",
-				  <&cam0_reg>,"gpio:0=", <&gpio>;
```

Comment:
This overriding of the GPIO controller is lost in the replacement, common definitions. This means that users won't be able to redirect from `&expgpio` to `&gpio`. I don't think there's any harm in setting this across all platforms even though it is unnecessary, but CM4 could get a dedicated implementation.

Reply:
Thank you. Reinstated.